logger/logger.go

```diff
@@ -0,0 +1,191 @@
+}
+
+// internal function to access logger (eck)
+func createLogger(fname string) *LocalLogger {
+	file, _ := os.OpenFile(fname, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0777)
+
+	return &LocalLogger{
+		filename: fname,
+		Logger:   log.New(file, "fedramp-templater: ", log.Lshortfile),
+	}
+}
+
+// log and print with given stack and format
+func internalPrint(stackLevels int, desired Level, format string, args ...interface{}) bool {
+	if Check(desired) {
+		var formattedStr = fmt.Sprintf(format, args...)
+		GetInstance().Output(stackLevels, fmt.Sprintf("%s %s\n", desired.Levels(), formattedStr))
+		return true
+	}
+	return false
+}
+
+// Check - should logging occur?
+func Check(desired Level) bool {
+	var envDebugStr = os.Getenv("DEBUG")
+	envDebugVal, err := strconv.Atoi(envDebugStr)
+	if err != nil {
+		return false
+	}
+	var actual = UNSUPP
+	switch envDebugVal {
+	case int(FATAL):
+		actual = FATAL
+	case int(ERROR):
+		actual = ERROR
+	case int(WARNING):
+		actual = WARNING
+	case int(INFO):
+		actual = INFO
+	case int(DEBUG):
+		actual = DEBUG
+	case int(TRACE):
+		actual = TRACE
+	default:
+		actual = UNSUPP
+	}
+	if actual.Levels() >= UNSUPP {
+		return false
+	}
+	return (actual.Levels() >= desired.Levels())
+}
+
+// Print - log and print
+func Print(desired Level, msg string) bool {
+	return internalPrint(2, desired, "%s", msg)
+}
+
+// Trace - log and print
+func Trace(msg string) bool {
+	return internalPrint(3, TRACE, "%s", msg)
+}
+
+// Debug - log and print
+func Debug(msg string) bool {
+	return internalPrint(3, DEBUG, "%s", msg)
+}
+
+// Info - log and print
+func Info(msg string) bool {
+	return internalPrint(3, DEBUG, "%s", msg)
+}
+
+// Warning - log and print
+func Warning(msg string) bool {
+	return internalPrint(3, WARNING, "%s", msg)
+}
+
+// Error - log and print
+func Error(msg string) bool {
+	return internalPrint(3, ERROR, "%s", msg)
+}
+
+// Fatal - log and print
+func Fatal(msg string) bool {
+	return internalPrint(3, FATAL, "%s", msg)
+}
+
+// Printf - log and print
+func Printf(desired Level, format string, args ...interface{}) bool {
+	return internalPrint(2, desired, format, args...)
+}
+
+// Tracef - log and print
+func Tracef(format string, args ...interface{}) bool {
+	return internalPrint(3, TRACE, format, args...)
+}
+
+// Debugf - log and print
+func Debugf(format string, args ...interface{}) bool {
+	return internalPrint(3, DEBUG, format, args...)
+}
+
+// Infof - log and print
+func Infof(format string, args ...interface{}) bool {
+	return internalPrint(3, DEBUG, format, args...)
+}
+
+// Warningf - log and print
+func Warningf(format string, args ...interface{}) bool {
+	return internalPrint(3, WARNING, format, args...)
+}
+
+// Errorf - log and print
+func Errorf(format string, args ...interface{}) bool {
+	return internalPrint(3, ERROR, format, args...)
+}
+
+// Fatalf - log and print
+func Fatalf(format string, args ...interface{}) bool {
+	return internalPrint(3, FATAL, format, args...)
+}
```

Comment:
While useful, this seems like the kind of functionality we can leverage an external package for, right? Just one more thing for us to maintain otherwise.

If we do want to keep it (for now, or longer), please add documentation about how to use it.